spec/sec-class-definitions-patch.html

```diff
@@ -243,7 +235,7 @@ <h1><ins>Runtime Semantics: ClassStaticBlockEvaluation</ins></h1>
       1. Let _innerContext_ be PrepareForClassStaticBlockEvaluation(_F_).
       1. Assert: _innerContext_ is now the running execution context.
       1. Perform ClassStaticBlockBindThis(_F_, _innerContext_).
-      1. Let _result_ be ClassStaticBlockEvaluateBody(_F_).
+      1. Let _result_ be ClassStaticBlockEvaluateBody(|ClassStaticBlockBody|).
```

Comment:
This is a bit weird, but we can address it later.

Reply:
How is this weird? We call AOs with parse nodes in other places in the spec. For example, the use of `IsAnonymousFunctionDefinition` here: 

https://tc39.es/ecma262/#sec-object-initializer-runtime-semantics-propertydefinitionevaluation
> 
![image](https://user-images.githubusercontent.com/3902892/104676023-73580500-569b-11eb-80dc-9d44bc690ef5.png)

Reply:
It's weird in that I'd like a sharper line dividing runtime SDOs and AOs. And agreed that this is an existing oddity, thus no need to act on it now.